src/repository/getUserDAO.js

```diff
@@ -0,0 +1,19 @@
+const { QueryCommand } = require('@aws-sdk/lib-dynamodb');
+const { TableName, UsernameIndex, runCommand } = require('../utilities/dynamoUtilities');
+
+async function queryByUsername(Username) {
```

Comment:
I would rename this to be getUserByUsername so that its more specific on what it's doing

Reply:
I wanted to avoid naming it that because it conflicts with the method name in the service layer. It also is more explicit about using a QueryCommand and not a GetCommand.